linkerd/app/core/src/metrics.rs

```diff
@@ -42,13 +45,24 @@ pub struct Metrics {
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
 pub struct ControlLabels {
     addr: Addr,
-    tls_status: TlsStatus,
+    server_id: tls::ConditionalServerId,
 }
 
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
-pub struct EndpointLabels {
-    pub direction: Direction,
-    pub tls_id: TlsStatus,
+pub enum EndpointLabels {
+    Inbound(InboundEndpointLabels),
+    Outbound(OutboundEndpointLabels),
+}
+
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+pub struct InboundEndpointLabels {
+    pub client_id: tls::server::ConditionalTls,
+    pub authority: Option<http::uri::Authority>,
```

Comment:
was kind of wondering if this should be on the `EndpointLabels` type, since both directions have an authority, but i realize that just complicates the `Into` impls a bunch so nvm.